code/controllers/shuttle_controller.dm

```diff
@@ -357,7 +357,7 @@ datum/emergency_shuttle/proc/process()
 
 		if(1)
 
-			if(timeleft == 5)
+			if(timeleft == 6 || timeleft == 5)
```

Comment:
Why not check if timeleft <= 6 and then put another var that prevents it from firing the sound again

Reply:
I considered doing that, but I tested this solution and it worked fine, so I told myself the less code lines the better.

Reply:
Doing less lines of code is what gave you this problem in the first place t b h

Reply:
touché